docs/components/modeler/web-modeler/model-your-first-diagram.md

```diff
@@ -0,0 +1,39 @@
+---
+id: model-your-first-diagram
+title: Model your first diagram
+description: "After you've created a BPMN diagram, you can start modeling it."
+---
+
+After you've created a BPMN diagram, you can start modeling it.
+
+We've preconfigured a diagram consisting of a start event. To convert it to something meaningful, append a **Task** to it and afterwards append an **EndEvent**:
+
+![add task](img/web-modeler-add-task.png)
+![add task](img/web-modeler-add-endevent.png)
+
+Each element has adjustable attributes. Use the properties panel on the right side of the page.
+
+Elements supporting different types can be reconfigured by clicking on the corresponding icon. In the following screenshot, a task has been added to the diagram. It can be converted to a service task, for example.
+
+![task configuration](img/web-modeler-new-diagram-with-configuration.png)
+
+Use the canvas tools on the bottom right corner to interact with your diagram
+
+1. Zoom in
```

Comment:
Let's keep this in mind when we review the Web Modeler docs in the future because I think we can do this in a slightly different way or work with the product design team to introduce these interaction features directly in the product.